db/seeds.rb

```diff
@@ -3,6 +3,92 @@
 
 exit if Rails.env.test?
 FactoryBot.create :user, first_name: 'David', last_name: 'Faulkenberry', staff: true, spire: '12345678@umass.edu', email: 'dfaulken@umass.edu', admin: true
+user = FactoryBot.create :user, first_name: 'Justin', last_name: 'Forgue', spire: '13579246@umass.edu', email: 'jforgue@umass.edu'
+department = FactoryBot.create :department, name: 'Bus'
+position = FactoryBot.create :position, department: department, name: 'Operator'
+FactoryBot.create :application_submission, data: [["Bus Application", nil, "heading", 1],
+                                                  ["Minimum Job Requirements for **Bus Driver**\r\n\r\n + You must have at least three semesters remaining as a UMass Student\r\n + You must be registered for the current or upcoming semester.\r\n + You must be willing to accept a work assignment of at least 15 hours per week during the semester.\r\n + You must be willing to work a minimum of 180 hours per semester for at least two full semesters after training is complete.\r\n + You must possess a driver's license that is valid in the United States.\r\n + You must be at least 18 years of age, and be able to communicate clearly in English.\r\n + You must be able to pass a NIDA-5 panel pre-employment drug screening and agree to participate in random, unannounced, NIDA-5 panel drug testing throughout your employment.\r\n + You must be willing to sign a legally binding contract that includes (but is not limited to) the above terms.\r\n + You must enjoy driving.\r\n", nil, "explanation", 81],
+                                                  ["I understand and meet the above minimum requirements for entering the UMass Transit bus driver training program.", "Yes", "yes/no", 2],
```

Comment:
I am unsure how to break these lines up without ruining the Markdown formatting .

Reply:
I would do this:

```ruby
FactoryBot.create :application_submission,
                  data: [['Bus Application', nil, 'heading', 1],
                         [<<~REQS, nil, 'explanation', 81],
                           Minimum Job Requirements for **Bus Driver**

                           +   You must have at least three semesters remaining
                               as a UMass Student.
                           +   You must be registered for the current or
                               upcoming semester.
                           +   You must be willing to accept a work assignment
                               of at least 15 hours per week during the
                               semester.
                           +   You must be willing to work a minimum of 180
                               hours per semester for at least two full
                               semesters after training is complete.
                           +   You must possess a driver's license that is
                               valid in the United States.
                           +   You must be at least 18 years of age, and be
                               able to communicate clearly in English.
                           +   You must be able to pass a NIDA-5 panel
                               pre-employment drug screening and agree to
                               participate in random, unannounced, NIDA-5 panel
                               drug testing throughout your employment.
                           +   You must be willing to sign a legally binding
                               contract that includes (but is not limited to)
                               the above terms.
                           +   You must enjoy driving.
                         REQS
                         ["Local address (street, apt., etc.)", "255 Governors Dr", "text", 3],
                         #ETC.

Reply:
You shouldn't need `\r`, and the actual newlines in the heredoc get included in the string.